hushline/settings/__init__.py

```diff
@@ -314,6 +324,9 @@ async def index() -> str | Response:
             pgp_key_percentage=pgp_key_percentage,
             directory_visibility_form=directory_visibility_form,
             default_forwarding_enabled=bool(current_app.config["NOTIFICATIONS_ADDRESS"]),
+            # Premium-specific data
+            is_premium_enabled=bool(current_app.config.get("STRIPE_SECRET_KEY", False)),
```

Comment:
This is a pattern we already use, but I wonder if somekinda `setup_g` pre-request function would be better for these types of derived configs. Or making a `stripe_ext` module with something like a `stripe.is_enabled` property or something. Don't need to change, just thinking aloud for future organization of the code.

Reply:
I actually refactored this already to make `is_premium_enabled` a global config available in all templates